signin.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>로그인</title>
+    <link rel="stylesheet" href="./src/singin.css" />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css"
+    />
+    <link rel="stylesheet" href="./src/reset.css" />
+  </head>
+  <body>
+    <div class="wrap">
+      <div class="header">
+        <div class="header1">
+          <a href="index.html"><img src="./images/signin-logo.png" /></a>
+          <div class="header1-text">
+            <p class="text1">회원이 아니신가요?</p>
+            <a href="signup.html"><p class="text2">회원 가입하기</p></a>
+          </div>
+        </div>
+      </div>
+      <div class="login-form">
+        <label for="email"><p class="email-text">이메일</p></label>
+        <input type="text" id="email" class="email-input" />
+        <label for="password"><p class="password-text">비밀번호</p></label>
+        <input type="text" id="password" class="password-input" />
+        <button class="login-btn">
+          <span class="login-btn-text">로그인</span>
+        </button>
+      </div>
+      <div class="footer">
+        <p class="footer-text">소셜 로그인</p>
+        <div class="footer-logo">
+          <a href="https://www.google.com"
+            ><img src="./images/ico_google.png"
+          /></a>
```

Comment:
```suggestion
          <a href="https://www.google.com">
            <img src="./images/ico_google.png" />
          </a>
```

요 포맷팅이 내려간건 이유가 있을까요?!